pkg/cmd/server/server.go

```diff
@@ -524,27 +550,28 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 		return clientmgmt.NewManager(logger, s.logLevel, s.pluginRegistry)
 	}
 
-	backupSyncController := controller.NewBackupSyncController(
-		s.veleroClient.VeleroV1(),
-		s.veleroClient.VeleroV1(),
-		s.sharedInformerFactory.Velero().V1().Backups(),
-		s.sharedInformerFactory.Velero().V1().BackupStorageLocations(),
-		s.config.backupSyncPeriod,
-		s.namespace,
-		s.config.defaultBackupLocation,
-		newPluginManager,
-		s.logger,
-	)
-	wg.Add(1)
-	go func() {
-		backupSyncController.Run(ctx, 1)
-		wg.Done()
-	}()
+	backupSyncControllerGetter := func() (controller.Interface, int) {
+		numControllerWorkers := 1
```

Comment:
I think this works. It might be a little bit clearer if we defined a simple struct along the lines of:
```go
type controllerRunInfo struct {
    controller controller.Interface
    numWorkers int
}
```
and return that from the anonymous functions - just slightly more self-documenting. I would also define a constant , `defaultControllerWorkers`, and just use that in all of these functions rather than declaring a new variable in each one. Can be overridden in individual functions later if needed, but right now it's not needed :)